src/connectors/aws_collect.py

```diff
@@ -1372,6 +1394,28 @@
             ],
         },
     },
+    'ebs.describe_snapshot':{
+      'response': {
+            'DescribeSnapshot': {
```

Comment:
the response has a single array `Snapshot` of items we want to load, so this code should be `'Snapshot': [...]`